src/main/resources/webapp/js/controllers.js

```diff
@@ -436,13 +436,12 @@
                         $scope.enrollInProgress = true;
                         $http.post('../ebodac/enrollSubject', subjectId)
                         .success(function(response) {
-                            var index = $scope.messages.push($scope.msg('ebodac.web.enrollment.enrollSubject.success'));
-                            hideMsgLater(index-1);
+                            motechAlert('ebodac.web.enrollment.enrollSubject.success', 'ebodac.web.enrollment.enrolledSubject');
                             $scope.refreshGrid();
                             $scope.enrollInProgress = false;
                         })
                         .error(function(response) {
-                            $scope.errors.push($scope.getMessageFromData(response));
+                            motechAlert('ebodac.web.enrollment.enrollSubject.error', 'ebodac.web.enrollment.error');
```

Comment:
You don't display the reason of this error, change this line to: motechAlert('ebodac.web.enrollment.enrollSubject.error', 'ebodac.web.enrollment.error', $scope.getMessageFromData(response));
Use this to other error messages as well.